tests/test_modeling_big_bird.py

```diff
@@ -0,0 +1,1337 @@
+            encoder_hidden_states,
+            encoder_attention_mask,
+        )
+
+    def test_retain_grad_hidden_states_attentions(self):
+        # bigbird cannot keep gradients in attentions when `attention_type=block_sparse`
+
+        if self.model_tester.attention_type == "original_full":
+            super().test_retain_grad_hidden_states_attentions()
+
+    @slow
+    def test_model_from_pretrained(self):
+        for model_name in BIG_BIRD_PRETRAINED_MODEL_ARCHIVE_LIST[:1]:
+            model = BigBirdForPreTraining.from_pretrained(model_name)
+            self.assertIsNotNone(model)
+
+    def test_model_various_attn_type(self):
+        config_and_inputs = self.model_tester.prepare_config_and_inputs()
+        for type in ["original_full", "block_sparse"]:
+            config_and_inputs[0].attention_type = type
+            self.model_tester.create_and_check_model(*config_and_inputs)
+
+    @unittest.skipIf(torch_device == "cpu", "Fast integration only compatible on GPU")
+    def test_fast_integration(self):
+        torch.manual_seed(0)
+
+        input_ids = torch.randint(
+            self.model_tester.vocab_size,
+            (self.model_tester.batch_size, self.model_tester.seq_length),
+            device=torch_device,
+        )
+        attention_mask = torch.ones((self.model_tester.batch_size, self.model_tester.seq_length), device=torch_device)
+        attention_mask[:, :-10] = 0
+        token_type_ids = torch.randint(
+            self.model_tester.type_vocab_size,
+            (self.model_tester.batch_size, self.model_tester.seq_length),
+            device=torch_device,
+        )
+
+        config, _, _, _, _, _, _ = self.model_tester.prepare_config_and_inputs()
+        model = BigBirdModel(config).to(torch_device).eval()
+
+        with torch.no_grad():
+            hidden_states = model(
+                input_ids, token_type_ids=token_type_ids, attention_mask=attention_mask
+            ).last_hidden_state
+            self.assertTrue(
+                torch.allclose(
+                    hidden_states[0, 0, :5],
+                    torch.tensor([-0.6326, 0.6124, -0.0844, 0.6698, -1.7155], device=torch_device),
+                    atol=1e-3,
+                )
+            )
+
+    def test_auto_padding(self):
+        self.model_tester.seq_length = 241
+        config_and_inputs = self.model_tester.prepare_config_and_inputs()
+        self.model_tester.create_and_check_for_auto_padding(*config_and_inputs)
+
+    def test_for_change_to_full_attn(self):
+        self.model_tester.seq_length = 9
+        config_and_inputs = self.model_tester.prepare_config_and_inputs()
+        self.model_tester.create_and_check_for_change_to_full_attn(*config_and_inputs)
+
+
+@require_torch
+@slow
+class BigBirdModelIntegrationTest(unittest.TestCase):
+    # we can have this true once block_sparse attn_probs works accurately
+    test_attention_probs = False
+
+    def _get_dummy_input_ids(self):
+        return torch.tensor(
```

Comment:
Deactivate styling here and put everything on big lines.

Reply:
You should put the following line before the array:
```
# fmt: off
```
and the following line after the array:
```
# fmt: on
```

You should also add the following at the end of the huge line:
```
  # noqa: E231
```

You can see an example of this in practice here:

https://github.com/huggingface/transformers/blob/d7b50ce46943a2b431a260a63166dc859745f0f6/tests/test_tokenization_tapas.py#L1188-L1190